docs/markdown/home.md

```diff
@@ -22,3 +22,5 @@ Thanks for visiting the documentation for [angularjs-portal](https://github.com/
 #### Integration with uPortal
 + Overview
 + Customizations/Setup
+
+[uw-frame docs]: http://uw-madison-doit.github.io/uw-frame/
```

Comment:
You can. Done, in 428af89.

Choice of Markdown link syntax, definitely an implementation detail I don't need to have an opinion about.

I think I reflexively went for the reference syntax because I find it makes the prose read better. I use reference link syntax quite a bit in my own Markdown writing.

Now giving this way more thought than I ought to, though, I did err here in introducing the reference syntax -- the existing `[angularjs-portal](https://github.com/UW-Madison-DoIT/angularjs-portal)` link uses inline syntax, and of course an important style consideration on an edit is to fit with the style of the thing being edited.